docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -115,7 +115,7 @@ You can sideload the add-in using the Teams Toolkit or in a command prompt, bash
 
 If the project was created with the Office Yeoman Generator and you don't want to use the Teams Toolkit, convert it using the following steps.
 
-1. In the root of the project, open a command prompt or bash shell and run the following command. This converts the manifest and updates the package.json to specify current tooling packages. The new unified manifest is in the root of the project and the old XML manifest is in a backup.zip file. For details about this command, see [Office-Addin-Project](https://www.npmjs.com/package/office-addin-project).
+1. In the root of the project, open a command prompt or bash shell and run the following command. This converts the manifest and updates the package.json to specify current tooling packages. The new unified manifest is in the root of the project and the old add-in only manifest is in a backup.zip file. For details about this command, see [Office-Addin-Project](https://www.npmjs.com/package/office-addin-project).
```

Comment:
Is there supposed to be a space here so that it refers to a generic backup zip file?

```suggestion
1. In the root of the project, open a command prompt or bash shell and run the following command. This converts the manifest and updates the package.json to specify current tooling packages. The new unified manifest is in the root of the project and the old add-in only manifest is in a backup .zip file. For details about this command, see [Office-Addin-Project](https://www.npmjs.com/package/office-addin-project).
```

Reply:
> Is there supposed to be a space here so that it refers to a generic backup zip file?

It's actually called backup.zip